Sources/App/Core/Search.swift

```diff
@@ -175,7 +183,18 @@ enum Search {
             .column(keywords)
             .column(hasDocs)
             .column(nullInt, as: levenshteinDist)
+            .column(ts_rank(vector: tsvector, query: tsquery), as: tsrankvalue)
             .from(searchView)
+            .from(plainto_tsquery(mergedTerms), as: tsquery)
```

Comment:
I've checked and neither additional "FROM" clause poses a problem WRT cardinatily, they're both (1).

They could actually just be select clauses. Is there reason you placed them in FROM so they're easier to use because the alternative would be to write out a bigger `ts_rank` clause that has them both inline?

If we pull "tsvector" into the search view it might make sense to do all this inline.